ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js

```diff
@@ -127,4 +127,21 @@ describe('ContextualMenuDecorator Specs', () => {
 
 		expect(contextualMenu).toHaveClass(expected);
 	});
+
+	test('should wrap content in Scroller component if it does not fit inside of popup', () => {
+		const moreItems = new Array(8).fill().map((i, index) => `Option ${index + 1}`);
+
+		render(
+			<Root>
+				<ContextualButton menuItems={moreItems} open>
+					Hello
+				</ContextualButton>
+			</Root>
+		);
+
+		const scrollerWrappedComponent = screen.getByRole('list').parentElement;
```

Comment:
```suggestion
		const scroller = screen.getByRole('list').parentElement;
```
I think the variable name is bit confusing. Could you please fix it?